apps/easypid/src/hooks/usePidCredential.tsx

```diff
@@ -226,12 +211,9 @@ export function getPidMetadataAttributesForDisplay(
 
   if (claimFormat === ClaimFormat.MsoMdoc) {
     const { holder, issuedAt, validUntil, type, ...metadataRest } = metadata
-    const { issuing_authority, issuing_country, issuance_date, expiry_date, ...remainingAttributes } =
-      attributes as PidMdocAttributes
+    const { issuance_date, expiry_date } = attributes as PidMdocAttributes
 
     return {
-      issuing_authority,
-      issuing_country,
```

Comment:
We need to rethink the metadata a bit. `issuing_country` and `issuing_authority` should be shows as that it will be shared I think? By adding them to metadata that is hidden unless you go to the overview page and then click show metadata. What do you think?

Reply:
Fair 👍  that's what i did now